app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -457,7 +457,7 @@ class EventDetailsFragment : Fragment() {
         // load location to map
         val mapClickListener = View.OnClickListener { startMap(event) }
 
-        if (!event.locationName.isNullOrEmpty() && event.longitude != null && event.latitude != null) {
+        if (!event.locationName.isNullOrEmpty() && event.longitude != null && event.longitude != 0.00 && event.latitude != null && event.latitude != 0.00) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [ComplexCondition - 5/4 - [loadEvent] at /src/app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt:460:13](https://app.codacy.com/manual/fossasia/open-event-android-general/pullRequest?prid=4850028)

Reply:
sure I thought it was too messy and will be pointed out... I will fix it 😇